src/types/product.interface.ts

```diff
@@ -0,0 +1,51 @@
+export interface IProduct {
```

Comment:
kind of bad practice to use capital "I" in front of interface name, please, use simply 'interface Product' and the same for the rest

Reply:
rename it